apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -54,20 +57,35 @@ export const fetchWorkflows = async (params: {
 
 export type TWorkflowById = z.output<typeof WorkflowByIdSchema>;
 
+export const BaseWorkflowDefinition = ObjectWithIdSchema.extend({
+  name: z.string(),
+  version: z.number(),
+  variant: z.string().default(WorkflowDefinitionVariant.DEFAULT),
+  contextSchema: z.record(z.any(), z.any()).nullable(),
+  documentsSchema: z.array(z.any()).optional().nullable(),
+  config: z.record(z.any(), z.any()).nullable(),
+  definition: z.record(z.any()),
+  transitionSchema: z
+    .array(
+      z.object({
+        state: z.string(),
+        schema: z.record(z.any()),
```

Comment:
This schema is invalid. `z.record` expects two arguments.
```suggestion
        schema: z.record(z.string(), z.unknown()),
```

Reply:
![image](https://github.com/ballerine-io/ballerine/assets/23522869/aaba14cf-1775-496c-a821-7cd3ee131840)

Reply:
https://zod.dev/?id=record-key-type

Reply:
![image](https://github.com/ballerine-io/ballerine/assets/23522869/14c795a3-3a5f-49ba-a9af-d588deb52519)

Reply:
Fixed